apps/sms/views/inbox/js/inbox.js

```diff
@@ -973,7 +984,21 @@ var InboxView = {
     }
   },
 
-  onDraftSaved: function inbox_onDraftSaved() {
+  onDraftDeleted: function inbox_onDraftDeleted(draft) {
```

Comment:
But in the long tern, maybe the best way is inbox view doesn't update the thread/draft change immediately, and it will listen to threads changes event for update.